### DISCUSSION TITLE
BKP6 coreuiv2

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#668] [Q&A (Help)]
I have almost completed the conversion from bkp5 to bkp6 with coreuiv2 theme but I have these two problems:
1. I don't see the footer even if it is set correctly
2. The navbar-toggler doesn't work for me even though I see that it saves the state in session
3. Also the toggle for menu item groups does'nt works
<img width="447" alt="bkp6-toggle-menu-and-items" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/8016df7b-5392-4c60-8477-1be385e7b8eb">

What can I check?

**Comment by promatik** [comment]
Hi @blondie63! Let's break it down 👌

> _I don't see the footer even if it is set correctly_

1) Please go to your layouts folder, make sure both `plain.blade.php` and `top_left.blade.php` have the footer section there.
You should have something like this there; https://github.com/Laravel-Backpack/theme-coreuiv2/blob/main/resources/views/layouts/plain.blade.php#L14
2) You can always use the [barryvdh debug bar](https://github.com/barryvdh/laravel-debugbar) to check what views are being load.

> _The navbar-toggler doesn't work for me even though I see that it saves the state in session_

The logic to handle the state of the sidebar is at `sidebar.blade.php`, so make sure you are including `'inc.sidebar'` in your blades, originally it is at `top_left.blade.php`.

> _Also the toggle for menu item groups doesn't works_

Regarding this, it's up to `@basset('https://unpkg.com/@coreui/coreui@2.1.16/dist/js/coreui.js')` to deal with that. So make sure it's being included somewhere (by default it's on `theme_scripts.blade.php`), and make sure you're using the correct css classes, take a look at the online demo if you need https://demo.backpackforlaravel.com/.

**Reply by blondie63** [reply]
Hi @promatik , i've checked and i've this:
plain.blade.php
```
    <footer class="app-footer sticky-footer">
        @include(backpack_view('inc.footer'))
    </footer>
```
top_left.blade.php
```
  @include(backpack_view('inc.sidebar'))

  <footer class="{{ backpack_theme_config('classes.footer') }}">
    @include(backpack_view('inc.footer'))
  </footer>
```

I'm not overrode footer.blade.php & theme_scripts.blade.php so i'm using theme default

Looking loaded views with Telescope i see these:

```
backpack.theme-coreuiv2::inc.theme_scripts
/vendor/backpack/theme-coreuiv2/resources/views/inc/theme_scripts.blade.php

backpack.theme-coreuiv2::inc.footer
/vendor/backpack/theme-coreuiv2/resources/views/inc/footer.blade.php

backpack.theme-coreuiv2::inc.sidebar
/vendor/backpack/theme-coreuiv2/resources/views/inc/sidebar.blade.php

backpack.theme-coreuiv2::layouts.top_left
/resources/views/vendor/backpack/theme-coreuiv2/layouts/top_left.blade.php
```

Also user menu does'nt works..
<img width="277" alt="bkp6-logout" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/9ca318e4-9a24-44d1-ab33-0d0c4a708790">

everything seems correct but i have the same problems
What else can I check ?

**Comment by blondie63** [comment]
Hi @promatik , have you seen my last msgs? :)

**Comment by blondie63** [comment]
I've found the problem !! Issues about bootstrap include !
